CHANGELOG.md

```diff
@@ -4,6 +4,34 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
 
 ## [unreleased][unreleased]
 
+## T55xx Downlink Protocol - 22 Jun 2019
```

Comment:
That's too much. The CHANGELOG.md is for the users, not for the developers. One or two lines under the existing "###Added" header should be enough to describe the new options.

Reply:
Fixed.

Reply:
Fixed some more. Want to add your name to the lines?

Reply:
Thanks, Happy to do what every protocol states/recommended.